Code/GraphMol/CIPLabeler/CIPLabeler.cpp

```diff
@@ -163,21 +163,35 @@ void label(std::vector<std::unique_ptr<Configuration>> &configs) {
   }
 }
 
+thread_local unsigned int remainingCallCount=0;
+
 }  // namespace
 
 void assignCIPLabels(ROMol &mol, const boost::dynamic_bitset<> &atoms,
-                     const boost::dynamic_bitset<> &bonds) {
+                     const boost::dynamic_bitset<> &bonds, unsigned int maxRecursiveIterations) {
+
+  if (maxRecursiveIterations != 0)
+    remainingCallCount  = maxRecursiveIterations;
+  else
+    remainingCallCount = UINT_MAX;  // really big - will never be hit
```

Comment:
We always include curly brackets:
```suggestion
  if (maxRecursiveIterations != 0) {
    remainingCallCount  = maxRecursiveIterations;
  } else {
    remainingCallCount = UINT_MAX;  // really big - will never be hit
  }
```

Reply:
I have been in places where the code style REQUIRES to not use unneeded braces.   Personally, I prefer to not put them in, but I don't care a lot.  I care more where they go (aligned braces), but I realize that the standards for this library is not going to change so ...